Materialy/Narzedzia/KIF.md

```diff
@@ -0,0 +1,49 @@
+# KIF - Keep It Functional - An iOS Functional Testing Framework
+KIF jest frameworkiem do tworzenia testów integracyjnych dla platformy iOS. W testach wykorzystywane są atrybuty accessibillity.
+KIF używa nieudokumentowanych API, dlatego kod wykorzystywany w testach nie powinien trafić do kodu produkcyjnego, bo aplikacja zostanie odrzucona w momencie weryfikacji przez Apple. 
+
+
+## Repozytorium
+
+* https://github.com/kif-framework/KIF
+    * Strona główna projektu
+* https://groups.google.com/forum/#!forum/kif-framework
```

Comment:
Grupa dyskusyjna to nie repozytorium :) Sugeruję inną nazwę sekcji zamiast "Repozytorium" lub inny podział.